Exiled.API/Features/Player.cs

```diff
@@ -1366,6 +1366,11 @@ public void ShowHint(string message, float duration = 3f)
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
 
+        /// <summary>
+        /// Show a HitMarker.
```

Comment:
```suggestion
        /// Shows a HitMarker.
```

as following most of the documentation.